src/Native/pkg/runtime.native.System.IO.Compression/win10/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <NativeFile Include="$(WinNativePath)..\..\NetCoreForCoreCLR\native\clrcompression.dll">
-      <TargetPath>runtimes/win10-$(PackagePlatform)-aot/lib/netcore50</TargetPath>
+    <NativeFile Include="$(WinNativePath)\clrcompression.dll">
```

Comment:
Think we want it in native since sometimes native assets are treated differently by NuGet than regular assets. 

We'll need to change it again anyways per the discussion here: https://github.com/dotnet/corefx/pull/9105#discussion_r65733433.